tg_bot/modules/welcome.py

```diff
@@ -140,6 +142,23 @@ def new_member(bot: Bot, update: Update):
             if sent:
                 sql.set_clean_welcome(chat.id, sent.message_id)
 
+        mute_user = sql.get_mute_pref(chat.id)
```

Comment:
You've increased the number of db accesses here. Merge this with the previous welcome setting better for checking welcome messages.

Reply:
ok, then a single query for both cases

Reply:
I have looked at ways of doing what you suggest but I do not want to alter what is already there. Any more suggestions on how to do it?